ENIGMAsystem/SHELL/Universal_System/Resources/polygon.h

```diff
@@ -1,5 +1,5 @@
 /** Copyright (C) 2021 Nabeel Danish
-/** 
+*** 
```

Comment:
To eliminate a warning.

Reply:
I think to match the comment style it is probably best to have ` **` instead of `***` (space instead of star)

Reply:
> I think to match the comment style it is probably best to have ` **` instead of `***` (space instead of star)

Oh, nevermind this, its been a while since I looked at this code and the comment style is indeed `***`. Oops.

Reply:
np.